woocommerce/pinterest-for-woocommerce
# Bugfix/handle plugin requirements

Closes #109 

### Changes proposed in this pull request

In addition to adding the minimum versions in the plugin's metadata in #121, this also adds an active check for those requirements and if one is not met the plugin won't initialize and show a notice. 

#### Screenshots

![image](https://user-images.githubusercontent.com/4016167/128253680-5abb0c21-ebab-4bea-b48f-f0ab21f34240.png)


### Detailed test instructions

1. Use WP-CLI to rollback to an older/unsupported version of WC, ( eg `wp plugin install woocommerce --version=5.0.0 --force`)
2. Check that you get the notice in wp-admin and that no menu entry is added for the plugin.
